File: app/src/main/java/me/akhilnarang/eztvapp/Network/ApiInterface.java
package me.akhilnarang.eztvapp.Network;

import me.akhilnarang.eztvapp.Model.EZTVModel;
import retrofit2.Call;
import retrofit2.http.GET;

public interface ApiInterface {

    @GET("get-torrents?limit=10&page=1")
    Call<EZTVModel> getTorrents();
}

File: app/src/main/java/me/akhilnarang/eztvapp/network/ApiInterface.kt
package me.akhilnarang.eztvapp.network

import me.akhilnarang.eztvapp.model.EZTVModel
import retrofit2.Call
import retrofit2.http.GET
import retrofit2.http.Query

interface ApiInterface {

    @GET("get-torrents")
    fun getTorrents(@Query("limit") limit: Int, @Query("page") page: Int): Call<EZTVModel>
}
